src/reporters.ts

```diff
@@ -180,19 +189,28 @@ export class HumanReporter extends QueryReporter {
             if (childO) {
               const childRecords = getArray(childO, 'records', []);
               // eslint-disable-next-line @typescript-eslint/no-explicit-any
-              childRecords.forEach((record: unknown) => {
+              childRecords.forEach((record: unknown, index) => {
```

Comment:
There can be more than one child record from a subquery, Account record can have more than one contact associated to it. For subquery records 2-n a synthetic record needs to be added to the results to produce output like 
```
force:data:soql:query -u myalias -q Select name, (Select FirstName, LastName from Contacts) from Account
 NAME                            CONTACTS.FIRSTNAME CONTACTS.LASTNAME 
 ─────────────────────────────── ────────────────── ───────────────── 
 Test                                                                 
 Sample Account for Entitlements Peter              Hale              
                                 null               Test              
Total number of records retrieved: 2.
```

Reply:
can we add a NUT to cover this case?